### ISSUE TITLE
Normalize headers to AWS in Sandbox

### ISSUE THREAD

**Issue by ryanblock** [architect/architect#698]
AWS munges header casing; express lowercases. We should ensure everything is mocked exactly to AWS.

**Comment by konsumer** [comment]
For reference (on lambda):
```headers:
   { accept: 'application/vnd.github.v3+json',
     'accept-encoding': 'gzip, deflate, br',
     'accept-language': 'en-US,en;q=0.5',
     Authorization: 'token 6c7d9c6673ab7b3ef254abbe85a7af783da9eb62',
     Host: 'river-pfe-staging.begin.app',
     referer: 'https://river-pfe-staging.begin.app/',
     'User-Agent':
      'octokit-request.js/5.3.2 Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:73.0) Gecko/20100101 Firefox/73.0',
     'X-Amzn-Trace-Id': 'Root=1-5e585ae1-9fb9f2ec7cf5b788e8666e3c',
     'X-Forwarded-For': '131.179.59.88',
     'X-Forwarded-Port': '443',
     'X-Forwarded-Proto': 'https' }
```

It looks like they capital-case the headers they are using for things (`X-*`, `Host`, `Authorization`, etc.)

Also, it should be notes express just lowercases `req.headers` but you can request `req.header('WHATEVER')` in any letter-case.

I made a test project [here](https://github.com/konsumer/express-header-test) if you want a quick way to explore it:

![Screenshot from 2020-02-28 13-37-21](https://user-images.githubusercontent.com/83857/75589455-a49df700-5a2f-11ea-9ba4-f0b0cac93d9f.png)

I think to emulate lambda, we'll need to wrap the function with something that injects mangled cases in `req.headers`, then calls function, rather than just run it as it is in express.

I really think it's a mistake to worry about letter-case at the app level, though, as it's non HTTP spec-compliant behavior. I like the idea of making everything the same as lambda in sandbox, but also maybe a simple normalizing util would be best, as the canonical way to do it, so it will work correctly if AWS changes how they letter-case the incoming headers, and it will work on other platforms.

**Comment by ryanblock** [comment]
It's def non-spec behavior, but it's also been a pain point for users, so I'd rather just remove this as a concern.

Here's what I've got thus far – this could be easily changed to support other platforms (or be disabled): https://github.com/architect/sandbox/pull/273/files

**Comment by ryanblock** [comment]
Released in Architect 6.2.4 and Sandbox 1.7.2!
